docs/UserGuide.md

```diff
@@ -78,12 +79,6 @@ Orange text     | Headings and subheadings of various size
 
 1. Refer to the [Features](#features) below for details of each command.
 
-<div markdown="block" class="alert alert-info">
-:exclamation: Try not to edit the **data file** directly!  
-* This may lead to invalid values for data fields. 
-* In the event of this, *ComputingConnection* will restart with an empty database! 
-</div>
-
```

Comment:
Dion, did you pull from the master branch before making changes? A lot of the new edits were deleted in this PR.